comm/MulticastSocket.cpp

```diff
@@ -211,31 +211,25 @@ bool MulticastSocket::autoSelectInterface()
     struct ifaddrs *ifa = NULL;
     getifaddrs(&ifAddrStruct);
 
-    // ignore list
-    std::set<std::string> ignore;
-    ignore.insert("lo");
-    ignore.insert("enp0s31f6"); // Falcons robots: the port on top-side of CPU-box, located most inward, is reserved for multiCam
-    
-    // priority list: assume ad-hoc cable connection takes precedence over wifi
-    std::set<std::string> prioritize;
-    prioritize.insert("enp3s0"); // Falcons robots
-    prioritize.insert("eth0"); // Falcons devlaptlops
-    
-    // TODO: make these lists configurable via XML?
-    
     // first select from priority list, then select whichever remains
     for (int priorityLoop = 0; priorityLoop <= 1; ++priorityLoop)
```

Comment:
This scores well on cyclomatic complexity...
Should this method have a unit test?